qa/Rakefile

```diff
@@ -0,0 +1,46 @@
+require "rspec"
+require "rspec/core/runner"
+require "rspec/core/rake_task"
+require_relative "vagrant-helpers"
+
+task :spec    => 'spec:all'
+task :default => :spec
+
+namespace :test do
+
+  desc "Generate a valid ssh-config"
+  task :ssh_config do
+    require "logstash/json"
```

Comment:
I do think this will work if you do `bundle exec` since the library will not be available anywhere.
You either need to use the core json or add a dependency on jrjackson.